src/Post.php

```diff
@@ -178,6 +179,17 @@ public function set_issues_links() {
 		$explore_page_id = $options['explore_page'] ?? '';
 		$categories      = get_the_category( $this->ID );
 
+		if ( PostPageCategoryLinks::is_active() ) {
+			$this->issues_nav_data = array_map(
```

Comment:
we could filter out `uncategorised` there (I just did that on [search filters](https://github.com/greenpeace/planet4-master-theme/pull/1671/files#diff-59c92d1bd26aebd2cddd3879fe484d83f59324074b2d50bad013fca4f6a1d60eR26-R28))